services/src/messaging/src/main/java/org/openkilda/messaging/payload/switches/PortConfigurationPayload.java

```diff
@@ -35,6 +35,7 @@
     @JsonProperty("status")
     private PortStatus status;
 
+    // TODO This property is to support port speed up-down 
```

Comment:
Please, could you add another TODO to the method "configurePort(PortConfigurationRequest)" as it's supposed to be responsible to applying the speed?

Reply:
Port speed task is under process. So I think no more TODO. 
On adding TODO at method level I have to either suppress check style violation or have to move TODO under <P> tag.